RevenueCatUI/Templates/Template4View.swift

```diff
@@ -136,7 +136,11 @@ struct Template4View: TemplateViewType {
                                   packageWidth: self.packageWidth,
                                   desiredHeight: self.packageContentHeight)
                 }
-                .buttonStyle(PackageButtonStyle(isSelected: isSelected))
+                .buttonStyle(PackageButtonStyle(
+                    // Reducing opacity would reveal the discount overlay
+                    // layers behind the button
+                    fadeDuringPurchases: false
+                ))
```

Comment:
Note that template 4 disables this behavior to avoid it looking like this:
<img width="328" alt="Screenshot 2023-10-05 at 10 37 28" src="https://github.com/RevenueCat/purchases-ios/assets/685609/3b6a3379-bc90-42ef-afab-81a596a922dd">